gno.land/pkg/gnoland/app.go

```diff
@@ -195,24 +199,58 @@ func EndBlocker(vmk vm.VMKeeperI) func(ctx sdk.Context, req abci.RequestEndBlock
 	}
 }
 
-func GuessGnoRootDir() string {
-	var rootdir string
+// XXX: all the method bellow should be removed in favor of
+// https://github.com/gnolang/gno/pull/1233
+var (
+	guessOnce sync.Once
+	gnoroot   string
```

Comment:
why do we need this? aren't we guessing the root only once to set the default flag value?

Reply:
Yes, but we are also using it for testing, so this method can be called a lot of time

Reply:
It could make sense for something in the critical path, like something called at each transaction. But here the code is more complex for a super light performance improvement that mostly (only?) happen in testing environment.

Remove complexity here please.